macOSにてulimitされない

## 概要

macOSにて`ulimit`がされずに, REを吐いてしまいます.
`ulimit -s 65532`をすると通ることを確認済みです.

## ログ

ライブラリのディレクトリにて`oj-verify run`を実行した際のログを以下に載せます.

<details>
<summary>ログ全文</summary>

```sh
INFO:onlinejudge_verify.config:no config file
WARNING:onlinejudge_verify.verify:failed to make the stack size unlimited
::warning ::failed to ulimit
INFO:onlinejudge_verify.verify:verify: /Users/morio/Documents/GitHub/cpplib/test/aoj/GRL_5_C.test.cpp
INFO:onlinejudge._implementation.logging:[x] problem recognized: AOJProblem.from_url('http://judge.u-aizu.ac.jp/onlinejudge/description.jsp?id=GRL_5_C'): https://onlinejudge.u-aizu.ac.jp/courses/library/5/GRL/5/GRL_5_C
INFO:onlinejudge_verify.verify:problem: https://onlinejudge.u-aizu.ac.jp/courses/library/5/GRL/5/GRL_5_C
INFO:onlinejudge_verify.languages.cplusplus:$ /usr/local/bin/g++ --std=c++17 -O2 -Wall -g -I /Users/morio/Documents/GitHub/cpplib -o .verify-helper/cache/079a3e64a6150f5593c305f3f9628bf1/a.out /Users/morio/Documents/GitHub/cpplib/test/aoj/GRL_5_C.test.cpp
In file included from /Users/morio/Documents/GitHub/cpplib/test/aoj/GRL_5_C.test.cpp:3:
/Users/morio/Documents/GitHub/cpplib/test/aoj/../../template/main.hpp:10: warning: ignoring #pragma region TEMPLATE [-Wunknown-pragmas]
   10 | #pragma region TEMPLATE
      | 
In file included from /Users/morio/Documents/GitHub/cpplib/test/aoj/GRL_5_C.test.cpp:3:
/Users/morio/Documents/GitHub/cpplib/test/aoj/../../template/main.hpp:118: warning: ignoring #pragma endregion  [-Wunknown-pragmas]
  118 | #pragma endregion
      | 
INFO:onlinejudge_verify.verify:$ oj test -c .verify-helper/cache/079a3e64a6150f5593c305f3f9628bf1/a.out -d .verify-helper/cache/079a3e64a6150f5593c305f3f9628bf1/test --print-input --tle 60
[*] 32 cases found
time: illegal option -- f
usage: time [-lp] command.
[!] GNU time is not available: time

[*] 00_sample_00.in
[x] time: 0.008904 sec
[+] AC

[*] 01_small_00.in
[x] time: 0.012699 sec
[+] AC

[*] 02_middle_00.in
[x] time: 0.004507 sec
[+] AC

[*] 02_middle_01.in
[x] time: 0.004316 sec
[+] AC

[*] 02_middle_02.in
[x] time: 0.004340 sec
[+] AC

[*] 02_middle_03.in
[x] time: 0.004652 sec
[+] AC

[*] 03_corner_00.in
[x] time: 0.004404 sec
[+] AC

[*] 03_corner_01.in
[x] time: 0.004370 sec
[+] AC

[*] 04_rand_00.in
[x] time: 0.004421 sec
[+] AC

[*] 04_rand_01.in
[x] time: 0.005065 sec
[+] AC

[*] 04_rand_02.in
[x] time: 0.004476 sec
[+] AC

[*] 04_rand_03.in
[x] time: 0.004505 sec
[+] AC

[*] 04_rand_04.in
[x] time: 0.004974 sec
[+] AC

[*] 04_rand_05.in
[x] time: 0.005277 sec
[+] AC

[*] 04_rand_06.in
[x] time: 0.005528 sec
[+] AC

[*] 04_rand_07.in
[x] time: 0.005573 sec
[+] AC

[*] 05_complete_00.in
[x] time: 0.004519 sec
[+] AC

[*] 05_complete_01.in
[x] time: 0.004745 sec
[+] AC

[*] 05_complete_02.in
[x] time: 0.004899 sec
[+] AC

[*] 05_complete_03.in
[x] time: 0.005145 sec
[+] AC

[*] 05_complete_04.in
[x] time: 0.006447 sec
[+] AC

[*] 05_complete_05.in
[x] time: 0.007958 sec
[+] AC

[*] 05_complete_06.in
[x] time: 0.015331 sec
[+] AC

[*] 05_complete_07.in
[x] time: 0.044669 sec
[+] AC

[*] 06_biased_00.in
[x] time: 0.120062 sec
[+] AC

[*] 06_biased_01.in
[x] time: 0.048213 sec
[-] RE: return code -11
input:
100000
1 1
1 2
1 3
1 4
1 5
1 6
1 7
1 8
1 9
1 10
1 11
1 12
1 13
1 14
1 15
1 16
1 17
1 18
1 19
... (199972 lines) ...
99990 99998
99991 99998
99992 99998
99993 99998
99994 99998
99995 99998
99996 99998
99997 99998
99998 99998
99999 99998

[-] WA
output:

expected:
0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
... (99970 lines) ...
99990
99991
99992
99993
99994
99995
99996
99997
99998
99998


[*] 06_biased_02.in
[x] time: 0.184418 sec
[+] AC

[*] 06_biased_03.in
[x] time: 0.168815 sec
[+] AC

[*] 07_large_00.in
[x] time: 0.185644 sec
[+] AC

[*] 07_large_01.in
[x] time: 0.186489 sec
[+] AC

[*] 07_large_02.in
[x] time: 0.210058 sec
[+] AC

[*] 07_large_03.in
[x] time: 0.203505 sec
[+] AC

[x] slowest: 0.210058 sec  (for 07_large_02.in)
[-] test failed: 31 AC / 32 cases
Traceback (most recent call last):
  File "/Users/morio/Library/Python/3.7/lib/python/site-packages/onlinejudge_verify/verify.py", line 113, in verify_file
    exec_command(command)
  File "/Users/morio/Library/Python/3.7/lib/python/site-packages/onlinejudge_verify/verify.py", line 45, in exec_command
    subprocess.check_call(command)
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['oj', 'test', '-c', '.verify-helper/cache/079a3e64a6150f5593c305f3f9628bf1/a.out', '-d', '.verify-helper/cache/079a3e64a6150f5593c305f3f9628bf1/test', '--print-input', '--tle', '60']' returned non-zero exit status 1.
::error file=test/aoj/GRL_5_C.test.cpp::failed to verify test/aoj/GRL_5_C.test.cpp
INFO:onlinejudge_verify.verify:verify: /Users/morio/Documents/GitHub/cpplib/test/yukicoder/399.test.cpp
INFO:onlinejudge_verify.verify:ignored
ERROR:onlinejudge_verify.verify:1 tests failed
ERROR:onlinejudge_verify.verify:failed: test/aoj/GRL_5_C.test.cpp
```
</details>

該当箇所は以下です(2~3行目).

```sh
WARNING:onlinejudge_verify.verify:failed to make the stack size unlimited
::warning ::failed to ulimit
```

## 実行環境

macOS Catalina 10.15.4 (19E287)

```sh
$ python3 --version
Python 3.7.7

$ oj --version
online-judge-tools 10.0.2 (+ online-judge-api-client 10.0.6)

$ pip3 show online-judge-verify-helper                  
Name: online-judge-verify-helper
Version: 4.9.1
Summary: UNKNOWN
Home-page: https://github.com/kmyk/online-judge-verify-helper
Author: Kimiyuki Onaka
Author-email: kimiyuki95@gmail.com
License: MIT License
Location: /Users/morio/Library/Python/3.7/lib/python/site-packages
Requires: toml, setuptools, pyyaml, online-judge-tools
Required-by: 
```

--- a/onlinejudge_verify/verify.py
+++ b/onlinejudge_verify/verify.py
@@ -120,9 +120,10 @@ def verify_file(path: pathlib.Path, *, compilers: List[str], tle: float, jobs: i
 
 def main(paths: List[pathlib.Path], *, marker: onlinejudge_verify.marker.VerificationMarker, timeout: float = math.inf, tle: float = 60, jobs: int = 1) -> VerificationSummary:
     try:
-        resource.setrlimit(resource.RLIMIT_STACK, (resource.RLIM_INFINITY, resource.RLIM_INFINITY))
+        _, hard = resource.getrlimit(resource.RLIMIT_STACK)
+        resource.setrlimit(resource.RLIMIT_STACK, (hard, hard))
     except:
-        logger.warning('failed to make the stack size unlimited')
+        logger.warning('failed to increase the stack size')
         print(f'::warning ::failed to ulimit')
 
     compilers = []

